doc/5.reference/file-help.pd

```diff
@@ -424,17 +424,16 @@ and returns the resolved path., f 69;
 #X obj 31 22 file which;
 #X text 114 22 - locate a file;
 #X text 127 65 symbol - file to locate using Pd's search-paths.;
-#X text 132 174 float - when there's an error creating the directory.
-;
 #X text 64 86 verbose <float> - set verbosity on or off., f 57;
-#X text 125 142 list - symbol path of located file and directory flag.
-;
 #X obj 7 208 cnv 2 550 2 empty empty ARGUMENTS: 8 12 0 13 #202020 #000000
 0;
 #X obj 7 234 cnv 1 550 1 empty empty flags: 8 12 0 13 #9f9f9f #000000
 0;
+#X text 98 143 list - symbol path of located file and directory flag.
+;
+#X text 85 174 symbol - the input symbol if file isn't located.;
 #X restore 418 15 pd reference;
-#X text 157 273 directory?;
+#X text 157 273 directory flag \, always 0 \, see notes below.;
```

Comment:
i don't like this.
the 2nd atom is a directory flag, that is `1` if the 1st atom is a directory - it is *not always* 0.

the fact that directories currently don't work is already documented. but the user should *always* be prepared that `[flag which]` might output a *1* here.

Reply:
> but the user should always be prepared that [flag which] might output a 1 here.

I see, so there are plans to implement it, right? And by saying the result is always 0 might give the idea that it may never in the future be different.

Reply:
well, fixed it!